charts/k10/k10/4.5.6/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  catalog.cattle.io/certified: partner
+  catalog.cattle.io/display-name: kasten/k10
+  catalog.cattle.io/release-name: k10
+apiVersion: v2
+appVersion: 4.5.6
+description: Kasten’s K10 Data Management Platform
+home: https://kasten.io/
+icon: https://docs.kasten.io/_static/kasten.png
```

Comment:
Icon URL Information is provided in the `Chart.yaml`. Currently, we are using [https://docs.kasten.io/_static/kasten.png](https://docs.kasten.io/_static/kasten.png). It has only `Kasten`. I will update the PR to use [https://docs.kasten.io/_static/kasten-logo-vertical.png](https://docs.kasten.io/_static/kasten-logo-vertical.png). This one uses `Kasten by Veaam`.